wordpress.org/public_html/wp-content/themes/pub/wporg-support/functions.php

```diff
@@ -39,6 +70,9 @@ function wporg_support_scripts() {
 	wp_enqueue_style( 'forum-wp4-style', get_stylesheet_uri(), [ 'dashicons' ], filemtime( __DIR__ . '/style.css' ) );
 	wp_style_add_data( 'forum-wp4-style', 'rtl', 'replace' );
 
+	wp_enqueue_style( 'wporg-parent-2021-style', 'https://developer.wordpress.org/wp-content/themes/wporg-parent-2021/build/style.css', [ 'wporg-global-fonts' ] );
```

Comment:
```suggestion
	wp_enqueue_style( 'wporg-parent-2021-style', get_theme_root_uri() . '/wporg-parent-2021/build/style.css', [ 'wporg-global-fonts' ] );
```

This would at least not rely on a separate site URL.

Reply:
Works great, thanks